salesforce_api/kafka_salesforce_integration/kafka_msgProducer/main.bal

```diff
@@ -0,0 +1,23 @@
+import ballerina/http;
```

Comment:
Instead of having package name as `kafka_msgProducer` follow the same practices of hyphens. So it is better to have as `kafka-message-producer`

Reply:
Updated in [bcf09ac](https://github.com/ballerina-guides/integration-samples/pull/31/commits/bcf09ac42e87543485854586312facacacd050cb)